### PR TITLE
Introduce a new directory for tasks related to structs

### PR DESCRIPTION
**Pull request by kech45**

### CODE REVIEW

**Review by HakanSunay — CHANGES_REQUESTED** [review]
The `README.md` file seems to be empty, please fill it in with some info.

**Comment by kech45** [comment]
> The `README.md` file seems to be empty, please fill it in with some info.

fixed.

File: structs/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This directory tracks the solutions of the problems, listed in https://github.com/vasilzahariev/IP-Practicum-Tasks/tree/main/Week%2012%20-%20Structsgi
```
Comment (HakanSunay) [inline structs/README.md]: nit: redundant comma

**Review by HakanSunay — CHANGES_REQUESTED** [review]

**Comment by HakanSunay** [comment]
LGTM 🚢

**Comment by kech45** [comment]
closes #1

**Pull request merged**